mgmt_cli_test.py

```diff
@@ -1550,3 +1551,88 @@ def test_restore_data_without_manager(self):
 
         self.test_restore_from_precreated_backup(snapshot_name, restore_outside_manager=True)
+
+
+class ManagerBackupRestoreConcurrentTests(ManagerTestFunctionsMixIn):
+    def test_backup_benchmark(self):
+        def report_to_argus(data, label):
+            send_manager_benchmark_results_to_argus(
+                argus_client=self.test_config.argus_client(),
+                result=data,
+                sut_timestamp=mgmt.get_scylla_manager_tool(
+                    manager_node=self.monitors.nodes[0]).sctool.client_version_timestamp,
+                row_name=label,
+            )
+
+        def create_backup(label):
+            mgr_cluster = self.ensure_and_get_cluster(
+                mgmt.get_scylla_manager_tool(manager_node=self.monitors.nodes[0]))
+
+            backup_start_time = time.time()
+            task = mgr_cluster.create_backup_task(location_list=self.locations, rate_limit_list=["0"])
+            backup_task_status = task.wait_for_uploading_stage(timeout=200000)
+            assert backup_task_status, "Backup has failed!"
+            backup_end_time = time.time()
+            InfoEvent(message=f'Backup without upload took: {backup_end_time - backup_start_time:.2f}s.').publish()
+            upload_start_time = time.time()
+            backup_status = task.wait_and_get_final_status(timeout=200000)
+            upload_end_time = time.time()
+            assert backup_status == TaskStatus.DONE, "Backup upload has failed!"
+            InfoEvent(
+                message=f'Backup upload upload took: {upload_end_time - upload_start_time}.').publish()
+            InfoEvent(
+                message=f'Backup total time is: {(backup_end_time - backup_start_time) + (upload_end_time - upload_start_time)}.').publish()
+            backup_report = {
+                "backup time": int(backup_end_time - backup_start_time),
+                "upload time": int(upload_end_time - upload_start_time),
+                "total": int((backup_end_time - backup_start_time) + (upload_end_time - upload_start_time)),
+            }
+            report_to_argus(backup_report, label)
+            return task
+
+        def run_read_stress(label):
+            stress_queue = []
+
+            for command in self.params.get('stress_read_cmd'):
+                stress_queue.append(self.run_stress_thread(command, round_robin=True, stop_test_on_failure=False))
+            read_start_time = time.time()
+            for stress in stress_queue:
+                assert self.verify_stress_thread(cs_thread_pool=stress), "Read stress command"
+
+            read_end_time = time.time()
+            InfoEvent(message=f'Read stress duration: {read_end_time - read_start_time:.2f}s.').publish()
+            read_stress_report = {
+                "read stress time": int(read_end_time - read_start_time),
+            }
+            report_to_argus(read_stress_report, label)
```

Comment:
I'm not sure `report_to_argus` should be a part of these `run_read_stress` and `create_backup` methods.
At least, method names are missing this part

Reply:
renamed